fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexScrubber.java

```diff
@@ -221,8 +228,21 @@ public Builder setEntriesScanLimit(long entriesScanLimit) {
                 return this;
             }
 
+            /**
+             * Declare that the scrubbed index is a value index, even if it's type's name is different.
+             *
+             * Scrubbing dangling index entries only makes sense for value indexes, where the index entry points back to
+             * the record.
+             * Typically, this function is called to allow scrubbing of an index of a user-defined type. If called, it
```

Comment:
```suggestion
             * Typically, this function is called to allow scrubbing of an index of a user-defined index type. If called, it
```

This may be using "index" one too many times, but I think "index of a user-defined type" is ambiguous in that someone might interpret that to mean "index defined on a user-defined record type" (which is, well, essentially any index)

Reply:
👍